test/integration/next-image-legacy/default/pages/priority.js

```diff
@@ -15,8 +15,8 @@ const Page = () => {
       <Image
         priority
         id="basic-image-with-crossorigin"
-        crossOrigin="anonymous"
-        src="/test.jpg"
```

Comment:
The purpose of this test was to confirm that we don't preload the same image twice. By changing it to `/test.gif`, it is no longer testing the same thing

Reply:
Added an identical `Image` to test we only preload once.